### PR TITLE
expand and improve logsearch troubleshooting

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:
- correct sample script
- add docs on easier fix
- add explanation for fixes
- use easier-to-follow examples

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/better-logsearch-directions)

## Security Considerations
None

### CODE REVIEW

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 1]
```diff
@@ -158,32 +158,51 @@ bosh -d logsearch recreate ingestor/0
...
+1. try reloading the `discovery` tab - it should work within a minute or so.
+
+This works by forcing Kibana to reload the list of fields. Doing so makes it realise there's an inconsistent mapping
+on the problem field, so it queries Elasticsearch more smartly about this field.
+
+If that doesn't work, you can use the steps below to rename the field on the documents with a mismatched field type.
 
 1. Use your browser's dev tools to figure out what index is failing the search. Using the network tab, look at requests for the `_msearch` endpoint (they'll also have query params). 
-1. In the response, look at `responses[0]['_shards']['failures'][0]['reason']['reason']`, which should have a message like `Fielddata is disabled on text fields by default. Set fielddata=true on [app.some.field]`. In this case `app.some.field` is the field that broke.
+1. In the response, look at `responses[0]['_shards']['failures'][0]['reason']['reason']`, which should have a message like `Fielddata is disabled on text fields by default. Set fielddata=true on [app.data.cert.Expiration]`. In this case `app.cert.Expiratoin` is the field that broke.
```
Comment (ccostino) [inline _docs/ops/runbook/troubleshooting-logsearch.md]: Small typo, and possible omission of a property?

```suggestion
1. In the response, look at `responses[0]['_shards']['failures'][0]['reason']['reason']`, which should have a message like `Fielddata is disabled on text fields by default. Set fielddata=true on [app.data.cert.Expiration]`. In this case `app.data.cert.Expiration` is the field that broke.
```

File: _docs/ops/runbook/troubleshooting-logsearch.md [hunk 2]
```diff
@@ -158,32 +158,51 @@ bosh -d logsearch recreate ingestor/0
...
 1. run this command, changing the field name and date as appropriate (in two places in script.source, one place in query.boo.filter.exists.field).
    ```
-   curl -XPOST localhost/$INDEX_WITH_PROBLEM/_update_by_query?refresh -H 'content-type: application/json' -d '{
+   curl -XPOST localhost/logs-app-2020.05.04/_update_by_query?refresh -H 'content-type: application/json' -d '{
      "script": {
-       "source": "ctx._source.app.some.thing$TODAYS_DATE = ctx._source.app.some(\"thing\");",
+       "source": "ctx._source.app.data.cert.Expiration20200504 = ctx._source.app.data.cert.remove(\"Expiration\");",
```
Comment (ccostino) [inline _docs/ops/runbook/troubleshooting-logsearch.md:186]: Thank you, much clearer!

**Review by ccostino — COMMENTED** [review]
Thank you, @bengerman13!  I caught a typo, and what appears to be a missing property as well (but I'm not 100% sure).

**Review by soutenniza — APPROVED** [review]
The extra context helps so LGTM 👍

**Pull request merged**